toolchain/sem_ir/file.h

```diff
@@ -225,10 +225,11 @@ class File : public Printable<File> {
   auto reals() const -> const ValueStore<RealId>& {
     return value_stores_->reals();
   }
-  auto string_literals() -> StringStoreWrapper<StringLiteralId>& {
+  auto string_literals() -> StringStoreWrapper<StringLiteralValueId>& {
```

Comment:
While I'm hesitant regarding renaming this entity versus others (wondering if this should get precedent for a shorter name since it's used cross-phase), I do think this should now be `string_literal_values` because you're reserving "string literal" to mean other things.

Reply:
Done. (@josh11b and I had discussed this and we didn't feel strongly either way, but happy to make the change since it sounds like you do.)

Reply:
Thanks -- it makes me sigh a little if we're headed in this direction, but I think if "Value" needs to be appended to all the types then it should also be appended to the stores.